Jellyfin.Api/Helpers/MediaInfoHelper.cs

```diff
@@ -237,6 +237,12 @@ public class MediaInfoHelper
                     user.HasPermission(PermissionKind.EnableAudioPlaybackTranscoding));
             }
 
+            // If the user doesn't have access to transcoding or has disabled it, then ignore the bitrate limit of the client
+            if (!mediaSource.SupportsTranscoding)
```

Comment:
`mediaSource.SupportsTranscoding` is `true` for Live TV regardless of the transcoding permission.

The previous condition also had no effect on Live TV.